### ISSUE TITLE
TypeError: configPaths.join is not a function while bulding error using latest version 1.71.0

### ISSUE THREAD

**Issue by pratik-vii** [serverless/serverless#7736]
New version of serverless giving following error while building application.
command: `serverless deploy`

```
TypeError: configPaths.join is not a function
      at module.exports._.memoize.inputArray (/usr/local/lib/node_modules/serverless/lib/utils/resolveCliInput.js:35:65)
      at memoized (/usr/local/lib/node_modules/serverless/node_modules/lodash/lodash.js:10552:27)
      at new Serverless (/usr/local/lib/node_modules/serverless/lib/Serverless.js:42:34)
      at Object.<anonymous> (/usr/local/lib/node_modules/serverless/bin/serverless.js:74:20)
      at Module._compile (internal/modules/cjs/loader.js:778:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
      at Module.load (internal/modules/cjs/loader.js:653:32)
      at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
      at Function.Module._load (internal/modules/cjs/loader.js:585:3)
      at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
      at startup (internal/bootstrap/node.js:283:19)
      at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              10.20.1
     Framework Version:         1.71.0
     Plugin Version:            3.6.12
     SDK Version:               2.3.0
     Components Version:        2.30.10
```

**Comment by vinthedark** [comment]
I am getting a similar issue after the newer update.

**Comment by medikoo** [comment]
Looks as regression introduced with https://github.com/serverless/serverless/pull/7728
 (cc @AhmedFat7y )

Patching a fix

**Comment by prashantramangupta** [comment]
Yes, the new version of serverless is giving above error. Prefer using the version 1.70.1 until it is fixed.

**Comment by medikoo** [comment]
Fix published with v1.71.1
